dmv/dmv.js

```diff
@@ -1,7 +1,44 @@
 function DMV(agents) {
-};
+    this.agents = agents,
+    this.customers = [];
+}
 
 DMV.prototype.customersInLine = function () {
+    return this.customers
+        .filter(customer=>!customer.agent)
+        .map(customer=>customer.customer);
+};
+DMV.prototype.enter = function (customer) {
+    return this.customers.push({"customer": customer});
+};
+
+DMV.prototype.currentCustomerFor = function (agent) {
+
+    let customersServed = this.customers
+        .filter(customer=> customer.agent===agent);
+        if(customersServed.length===0){
+            return null;
+        }
+        return customersServed[0].customer;
```

Comment:
👍 On array methods again, see if you can simplify the code by using `.find()` instead of `.filter()`